migrator/clone/rocksdb/db_clone_manager_impl.go

```diff
@@ -327,27 +328,46 @@ func (d *dbCloneManagerImpl) GetDirName(cloneName string) string {
 func (d *dbCloneManagerImpl) DecommissionRocksDB() {
 	log.Info("DecommissionRocksDB")
 	for k := range d.cloneMap {
-		path := d.getPath(k)
-		if exists, err := fileutils.Exists(path); err != nil {
+		dirPath := d.getPath(k)
```

Comment:
Decomission rocksdb should only happens when we have at least 1 postgres versions before and are attempting to upgrade to another. Why changing this could help in the case you described?

Reply:
Because there are things like bolt and bleve and telemetry that still reference current and at least started by central.  In fact central puts a new Rocks in there.  We discussed this a little when I first put this method in but unwinding all that other stuff was going to become a bigger challenge than it looked on the surface.  So we decided we would take out the rest of the stuff when ROX-9882 is done.  At that time we can really remove current, .db-init, and whatever else is there.

We don't have to worry about being confused when we see the rocksdb clone because when this is executed the migration_version.yaml is removed.  So even though we will see a current rocksdb, its version will be 0 and thusly ignored in favor of Postgres.

Reply:
I'd like to go through everything and make sure we stop referencing these things in central now, but I don't think we have the time for that.

Reply:
Old versions add new rocks if it cannot find a valid current clone. In the scenario you described in test steps, it only upgrade to one version of Postgres, that means Central PVC has a valid set of databases including bolt, bleve and rocks. 

The scenario here is after Postgres migrated twice, if the databases are decomissioned, rolling back to an old rocksdb version would create new databases. Of course that is not supported but we still need to give it a way to recover :).